### DISCUSSION TITLE
Deploying to Vercel with Prisma ORM

### DISCUSSION THREAD

**Discussion by kayandra** [vercel/next.js#42096] [Help]
I tried deploying next 13 with prisma to vercel and I got this error.

![Screenshot 2022-10-28 211747](https://user-images.githubusercontent.com/5002506/198749558-6bfb10c3-5043-4625-a09a-8aaf06781118.png)

The prisma schema is missing from the build. I don't know what that's happening, my folder structure looks like this

![Screenshot 2022-10-29 001730](https://user-images.githubusercontent.com/5002506/198749489-b8d5d8b8-7a36-493d-8c2b-f690a223b30f.png)

Has anyone run into this issue before?

**Comment by lukecarr** [comment]
Getting the same issue but during development, so don't think this is production build-specific.

**Reply by kayandra** [reply]
That's unlucky. Are you by chance in a monorepo setup? I read somewhere it could be one of the issues.

But since I'm not using a monorepo, it's a weird bug for sure.

**Reply by lukecarr** [reply]
Yeah, I'm using PNPM workspaces with Turborepo.

**Reply by kayandra** [reply]
I will post an answer here if I find a solution.

**Reply by kayandra** [reply]
@lukecarr here's what works for me atm https://github.com/prisma/prisma/issues/16117#issuecomment-1304396473

**Comment by shadcn** [comment]
I created an issue on the Prisma repo: https://github.com/prisma/prisma/issues/16117

Some findings:

I've tried a few ways to try and reproduce this and **this only happens when you make the page dynamic**.

A few ways I've been able to reproduce this:

1. Using the export const revalidate = 0 segment config on a page
2. Calling headers() from "next/headers" inside the page.

**Reply by kayandra** [reply]
hahahha, i came here to link your issue to this as a way to keep track. 

I switched to netlify for hosting and there's no issues, I think this is a vercel problem.

**Reply by shadcn** [reply]
Tried Netlify. No luck. Does it work with dynamic pages?

**Reply by kayandra** [reply]
![Screenshot 2022-11-04 195833](https://user-images.githubusercontent.com/5002506/200054368-6fe71602-7382-41af-98b5-473c30d8dd09.png)

Yes!

**Comment by leerob** [comment]
Hey! This should be fixed as of 13.0.2. Please let me know if not.

**Reply by shadcn** [reply]
@leerob Tried `13.0.2`. No luck. Same issue.

**Reply by kayandra** [reply]
Updated to 13.0.2. Did a fresh deploy and still the same thing

![Screenshot 2022-11-05 081311](https://user-images.githubusercontent.com/5002506/200107680-1cdfd36c-01cb-4404-ba2a-4ba15f20fb0f.png)

**Reply by kayandra** [reply]
This is what currently works https://github.com/prisma/prisma/issues/16117#issuecomment-1304396473
